packages/admin/cms-admin/src/form/file/upload.ts

```diff
@@ -35,10 +42,15 @@ function uploadOrReplaceByFilenameAndFolder<ResponseData>({
     const formData = new FormData();
     formData.append("file", data.file);
     formData.append("scope", JSON.stringify(data.scope));
-    if (data.importSourceId && data.importSourceType) {
+
+    if (replace === false && data.importSourceId && data.importSourceType && !data.file.importSource) {
```

Comment:
Why the `replace === false` check?

Reply:
should it be possible to change the importSource after the initial upload?

think if you ever require to swap to another import source you would rather write a database migrations to exchange importSourceId and Type

else the importSource should stay fixed so would remove the possibility to change it here in the first place to prevent accidents

Reply:
The reason why we save the import source type and id is to maintain a connection to the source asset. This connection can be used to sync the data with the source system (e.g. license data, alt text).

What if a file that was imported from a third-party system is 

- replaced by a file uploaded from the local machine
- replaced with another file imported from the third-party system (because they have the same name)

In both cases, the connection to the original imported file should be lost because it's not that file anymore. I don't want the license data or alt text to be synced because it might not apply anymore.

So I think we should update the import source also on replace (if it's replaced by a local file, it should be set to null).

Reply:
Ok. Yeah makes sense.

changed it back to enable override on replace: 6618afd1a46c17f37d2b2141f7665803200f9819

---- 
when the topic of syncs occurs, it will be interesting what further adjustments we might need i.e. in regards how the sync should handle manually overwritten files. if it will require a flag on the file or the folder/-structure the sync works within might be made read-only.

Reply:
> when the topic of syncs occurs, it will be interesting what further adjustments we might need i.e. in regards how the sync should handle manually overwritten files. if it will require a flag on the file or the folder/-structure the sync works within might be made read-only.

Yes true, we'll look into that later